libs/utils/env.py

```diff
@@ -441,6 +448,8 @@ def _init_target(self, force = False):
 
         elif self.conf['board'] != 'UNKNOWN':
             # Initilize from platform descriptor (if available)
+            self._log.error('Warning: UNKNOWN board type detected!!')
```

Comment:
Given that these are warnings (and they should be) they should be logged as a warnings with `self._log.warning`. The use of `Warning:` at the beginning of the message strings bring no useful information in this case.

Also, I would propose a rewording of the second message, maybe as "Execution on this target may have unexpected results".